webapp/src/components/sidebar_right/sidebar_right.jsx

```diff
@@ -148,6 +148,10 @@ export default class SidebarRight extends React.PureComponent {
                             complete={this.props.actions.complete}
                             accept={this.props.actions.accept}
                             bump={this.props.actions.bump}
+                            edit={(issue, e)=>{
```

Comment:
Why not follow the same pattern as the other actions? (bump, accept, complete...)

Reply:
Cannot use the same pattern because on edit two actions are getting called. Instead changed it to a method `edit`